src/test/java/com/github/tomakehurst/wiremock/admin/tasks/HealthCheckTaskTest.java

```diff
@@ -46,5 +47,6 @@ public void healthy() {
         response.getStatusMessage(),
         equalTo(response.getReponseBody().asJson().get("message").asText()));
     assertThat(response.getReponseBody().asJson().get("status").asText(), is("healthy"));
+    assertThat(response.getReponseBody().asJson().get("version").asText(), Matchers.notNullValue());
```

Comment:
This all looks great. We keep a [version.properties](https://github.com/wiremock/wiremock/blob/master/src/test/resources/version.properties) file in the test classpath so we can actually validate the specific verion returned here - `is("X.X.X")`

Reply:
@leeturner the change has been made in sha: 88fb3b6dbdd700d4cb807f58c0eb825b8547b0d8